src/nav/navSelectors.js

```diff
@@ -76,7 +76,7 @@ export const getInitialNavState = createSelector(
   getUsersById,
   (nav, accounts, auth, usersById) => {
     if (!auth.apiKey) {
-      return getStateForRoute(accounts && accounts.length > 1 ? 'account' : 'realm');
+      return getStateForRoute(accounts && accounts.length > 1 ? 'account' : 'realmgate');
```

Comment:
So, take another look at the existing logic here -- that's `> 1`, not `>= 1`. So your change applies when there are <= 1 accounts, not no accounts. That's the cause of the two `jest` failures.

I'm not quite sure why we drop the user at RealmScreen if there is 1 account; that doesn't seem very helpful. @borisyankov, do you have more context you can add for that?